src/miral/basic_window_manager.cpp

```diff
@@ -1280,7 +1280,7 @@ void miral::BasicWindowManager::place_attached_to_zone(
     }
 
     default:
-        fatal_error("BasicWindowManager::place_maximized() called for window not in a maximized or attached state");
+        log_error("BasicWindowManager::place_maximized() called for window not in a maximized or attached state");
```

Comment:
Preexisting, but the error message is clearly incorrect: the function is `BasicWindowManager::place_attached_to_zone()`